code/modules/jobs/job_types/church.dm

```diff
@@ -266,6 +273,39 @@
 	uniform = pick(
 		/obj/item/clothing/under/f13/settler,
 		/obj/item/clothing/under/f13/brahminm,
-		/obj/item/clothing/under/f13/machinist,
-		/obj/item/clothing/under/f13/lumberjack,
-		/obj/item/clothing/under/f13/roving)
+		/obj/item/clothing/under/f13/raiderharness,
+		/obj/item/clothing/under/f13/rag,
+		/obj/item/clothing/under/f13/raiderrags)
+
+/datum/outfit/loadout/neophyte/enforcer
+	name = "Enforcer"
+	backpack_contents = list(
+		/obj/item/melee/classic_baton = 1,
+		/obj/item/restraints/handcuffs/cable = 1,
+		/obj/item/restraints/legcuffs/bola = 1
+		)
+
+/datum/outfit/loadout/neophyte/healer
+	name = "Healer"
+	backpack_contents = list(
+		/obj/item/book/granter/trait/lowsurgery = 1,
+		/obj/item/stack/medical/bruise_pack = 1
+		)
+
+/datum/outfit/loadout/neophyte/sanitation
+	name = "Janitor"
+	backpack_contents = list(
+		/obj/item/melee/classic_baton = 1,
```

Comment:
I think you missplaced some items in the Janitors loadout.

Reply:
RIght, I fixed that